.github/workflows/Push.yml

```diff
@@ -15,11 +13,11 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       repo_owner: ${{ steps.repo_owner.outputs.lowercase }}
-      os_matrix: "{\"os_version\":[\"bullseye\",\"buster\",\"ubuntu16\",\"ubuntu18\",\"ubuntu20\"]}"
+      os_matrix: "{\"os_version\":[\"debian11\",\"debian10\",\"ubuntu20\"]}"
```

Comment:
I wasn't attending the last meetings - did you agree on removing the images for ubuntu16 and ubuntu18? Renaming bullseye and buster is fine for me.

Reply:
The Renaming and deprecation of old images was discussed with the new tagging scheme in the last meeting. 

https://github.com/hyperledger/indy-node-container/pull/119
https://wiki.hyperledger.org/pages/viewpage.action?pageId=78021180

Reply:
We should not remove the images just now. We are just issuing a warning that we will remove them soon https://github.com/hyperledger/indy-node-container/pull/119/files

Reply:
And We should definetly keep at least one stable ubuntu image. The ubuntu20 image is only for testing currently

Reply:
okay will fix it.